README.md

```diff
@@ -54,6 +54,11 @@ Community discussions happen primarily on the [dev mailing list][dev-list] or on
 
 Iceberg is built using Gradle 5.2.1.
 
+* To invoke a build and run tests, run: ./gradlew build
```

Comment:
This will look nicer if you use code formatting (backticks):
```
* To invoke a build and run tests, run: `./gradlew build`
```
I'd also simplify it by removing the *", run"* part on all of these:
```
* To invoke a build and run tests: `./gradlew build`
```
It will look like this when rendered:
* To invoke a build and run tests: `./gradlew build`

Reply:
I agree.

Reply:
Kk. I will update it.